package_management/package_manager.py

```diff
@@ -62,6 +65,19 @@ def query(self, name: str) -> Optional[PackageInfo]:
         else:
             return None
 
+    def remove_package_sync(self, package_name: str, package_version: str):
+        validate_package_name(package_name)
+        validate_package_version(package_version)
+
+        package_version_dir_path = self._paths_util.get_package_version_dir_path(package_name, package_version)
+        try:
+            print('Removing package ' + package_name + ' in version ' + package_version);
+            shutil.rmtree(package_version_dir_path)
+            with self._package_infos_lock:
+                self._remove_version_to_package_info(package_name, package_version)
+        except OSError as err:
+            logging.info(f'Successfully removed package: {fullname(package_name, package_version)}')
```

Comment:
Why success info on Exception? Assumption that file doesn't exist and therefore it's already deleted?